lib/datadog/tracing/contrib/rails/log_injection.rb

```diff
@@ -27,21 +27,6 @@ def add_as_tagged_logging_logger(app)
             )
             false
           end
-
-          def datadog_trace_log_hash(correlation)
-            {
-              # Adds IDs as tags to log output
-              dd: {
-                # To preserve precision during JSON serialization, use strings for large numbers
-                trace_id: correlation.trace_id.to_s,
-                span_id: correlation.span_id.to_s,
-                env: correlation.env.to_s,
-                service: correlation.service.to_s,
-                version: correlation.version.to_s
-              },
-              ddsource: ['ruby']
-            }
-          end
```

Comment:
This left me scratching my head a bit so I went digging to understand why this was still here and unused and here's what I learned using `git log -p -S datadog_trace_log_hash` (aka "git pickaxe"):

* This method was originally added in https://github.com/DataDog/dd-trace-rb/pull/1157, when the lograge integration was effectively included "inline" inside this file
* Then in https://github.com/DataDog/dd-trace-rb/pull/1555 it was moved away for a number of reasons (which are throughly documented in the PR description, thanks Eric!), but this was left behind and became unused since then.

Reply:
Sounds correct, I did the same git history digging, should have left a comment, sorry.